play.pokemonshowdown.com/src/battle-tooltips.ts

```diff
@@ -830,8 +830,9 @@ export class BattleTooltips {
 			genderBuf = ` <img src="${Dex.fxPrefix}gender-${gender.toLowerCase()}.png" alt="${gender}" width="7" height="10" class="pixelated" /> `;
 		}
 
-		let name = BattleLog.escapeHTML(pokemon.name);
-		if (pokemon.speciesForme !== pokemon.name) {
+		let ignoreNicks = this.battle.ignoreNicks || this.battle.ignoreOpponent;
```

Comment:
Does `ignoreOpponent` usually also apply `ignoreNicks`? I can see arguments for both approaches. I think because `ignoreNicks` is a separate setting, I would default to no.

Reply:
Looks like yes. Unless someone objects, I'm going to change that, but that's out of scope for this PR.